website/content/docs/ecs/terraform/install.mdx

```diff
@@ -421,15 +423,15 @@ module "web" {
 ## Configure the bind address
 
 To ensure that your application only receives traffic through the service mesh,
-you must change the address that your application is listening on to only the loopback address. The loopback address is also called `localhost`, `lo`, and `127.0.0.1`.
+you must change the address that your application listens on to the loopback address. The loopback address is also called `localhost`, `lo`, and `127.0.0.1`. 
 Binding to the loopback address allows the sidecar proxy running in the same task to only make requests within the service mesh.
 
-If your application is listening on all interfaces, e.g., `0.0.0.0`, then other
+If your application is listening on all interfaces, such as `0.0.0.0`, then other
 applications can call it directly, bypassing its sidecar proxy.
 
 Changing the listening address is specific to the language and framework you're
-using in your application. Regardless of which language/framework you're using,
-it is a good practice to make the address configurable via environment variable.
+using in your application. Regardless of which language or framework you're using,
+it is a good practice to use the environment variable to configure the address.
 
 The following examples demonstrate how to bind the loopback address in golang and Django (Python):
```

Comment:
The examples feel like a bit of a contradiction to me. We state above that the bind address should be configurable by means of an environment variable, and then we provide the reader with examples of how to hard-code the bind address :thinking: 

This is pre-existing but I thought I'd call it out. It's totally OK with me to consider this comment out of scope for this round of updates.

Reply:
What is your suggestion? Should we remove them? Can you provide updated examples?